server/src/app.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import logger from 'morgan';
+
+import { errHandler } from './helpers';
+import { postRouter, signInRouter, signupRouter } from './routes';
+import { likeRouter } from './routes/likeRouts';
+
+const app = express();
+
+// MIDDLEWARE
+app.use(express.json());
+app.use(logger('dev'));
```

Comment:
I don't know much about `morgan`, but it seems like a logging middleware, so we can't use it for manual logging correct? if so I think we'll need to use something different, or even cook up our own thing here. I'd like to not have to do `console.log/error/warn` and instead use the same logger with the same format for consistency.